aten/src/TH/generic/THTensor.cpp

```diff
@@ -287,7 +287,7 @@ THTensor *THTensor_(newView)(THTensor *tensor, THLongStorage *size)
 }
 
 /* Resize */
-void THTensor_(resize)(THTensor *self, THLongStorage *size, THLongStorage *stride)
+void THTensor_(resizeLegacy)(THTensor *self, THLongStorage *size, THLongStorage *stride)
```

Comment:
Deserves a comment on top of this function saying exactly what it means to be "legacy"